fastboot-server/headers.js

```diff
@@ -21,7 +21,8 @@ module.exports = function (req, res, next) {
 	if (req.headers['fastly-ssl']) {
 		const cspPolicy = 'default-src https: \'self\' data: blob:; ' +
 			'script-src https: \'self\' data: \'unsafe-inline\' \'unsafe-eval\' blob:; ' +
-			'style-src https: \'self\' \'unsafe-inline\' blob:; ';
+			'style-src https: \'self\' \'unsafe-inline\' blob:; '+
+			'frame-ancestors https://*.' + config.servicesDomain + ';';
```

Comment:
for sure we did not mean to use `config.servicesDomain` here. This points to the services domain like services.wikia.com